config/config.go

```diff
@@ -134,6 +134,9 @@ type ReporterConfig struct {
 	// Password instructs reporter to include a password for basic http authentication when sending spans to
 	// jaeger-collector. Can be set by exporting an environment variable named JAEGER_PASSWORD
 	Password string `yaml:"password"`
+
+	// Headers instructs the reporter to add these headers to the http request when reporting spans
+	Headers map[string]string `yaml:"headers"`
```

Comment:
nit: HTTPHeaders, and `http_headers`. Comment should mention that this field takes effect only when using HTTPTransport by providing `CollectorEndpoint `

Reply:
Yup, I like that suggestion, change made.